### ISSUE TITLE
Need to add Diamond developer and logo in relevant places (Trac #913)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#996]
In recognition that Tim has joined the development team we need to add his name, and the Diamond logo, in the relevant places (About, webpages, etc).

Migrated from http://trac.sasview.org/ticket/913
```json
{
    "status": "closed",
    "changetime": "2017-04-04T16:28:02",
    "_ts": "2017-04-04 16:28:02.942479+00:00",
    "description": "In recognition that Tim has joined the development team we need to add his name, and the Diamond logo, in the relevant places (About, webpages, etc).",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2017-04-04T12:07:59",
    "component": "SasView",
    "summary": "Need to add Diamond developer and logo in relevant places",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.2.0",
    "owner": "smk78",
    "type": "defect"
}
```

**Comment by smk78** [comment]
Trac update at `2017/04/04 15:35:41`: **smk78** commented:

Adjusted rendered size of footer images in css to create space, then added Diamond logo. Fixes webpages.

**Comment by smk78** [comment]
Trac update at `2017/04/04 16:28:02`: 
* **smk78** commented:

In changeset ce9e0b5e60c2ba92f798c8c5c8dde8fde673b201:

```text
#!CommitTicketReference repository="sasview" revision="ce9e0b5e60c2ba92f798c8c5c8dde8fde673b201"
Added DLS logo to guiframe images (for About box). Addresses #996
```


* **smk78** commented:

In changeset 1779e727216b236f9fc4d683fa29295ad44ed2ed:

```text
#!CommitTicketReference repository="sasview" revision="1779e727216b236f9fc4d683fa29295ad44ed2ed"
Incorporated DLS logo and URL, and added DLS to list of institutional
contributors. Closes #996
```


* **smk78** changed resolution from "" to "fixed"
* **smk78** changed status from "new" to "closed"
